ray-operator/main.go

```diff
@@ -133,6 +134,13 @@ func main() {
 		config.UseKubernetesProxy = useKubernetesProxy
 		config.DeleteRayJobAfterJobFinishes = os.Getenv(utils.DELETE_RAYJOB_CR_AFTER_JOB_FINISHES) == "true"
 		config.EnableMetrics = enableMetrics
+		preStopCommandListJson := os.Getenv(utils.PRE_STOP_COMMAND_LIST_JSON)
+		if preStopCommandListJson != "" {
+			umarshalErr := json.Unmarshal([]byte(preStopCommandListJson), &config.PreStopCommandList)
+			if umarshalErr != nil {
+				exitOnError(umarshalErr, "Failed to json parse preStopCommand")
```

Comment:
The error message is not actionable. 

```suggestion
				exitOnError(umarshalErr, "Failed to parse the environment variable PRE_STOP_COMMAND_LIST_JSON into JSON format.")
```

Reply:
Also, add an example in the error message.

Reply:
Probably `Failed to parse the environment variable PRE_STOP_COMMAND_LIST_JSON as a JSON array of strings` is more actionable.